compaction/compaction_manager.hh

```diff
@@ -275,6 +275,7 @@ private:
     timer<lowres_clock> _compaction_submission_timer = timer<lowres_clock>(compaction_submission_callback());
     static constexpr std::chrono::seconds periodic_compaction_submission_interval() { return std::chrono::seconds(3600); }
 
+    config _cfg;
```

Comment:
Since the whole config now lives on compaction manager, it's worth removing all the duplicates like `_compaction_sg`, `_maintenance_sg`, `_throughput_mbs`, `_static_shares` and `_available_memory`

Reply:
makes sense

Reply:
Done in v5